src/lib/core/components/filter/HistogramFilter.tsx

```diff
@@ -573,3 +575,15 @@ async function getHistogram(
     getRequestParams(studyId, filters, entity, variable, dataParams, rawConfig)
   );
 }
+
+// TODO [2021-07-10] - Use variable.precision when avaiable
+function formatStatValue(
+  value: string | number,
+  type: HistogramVariable['type']
+) {
+  return type === 'date'
+    ? value
+    : Number(value).toLocaleString(undefined, {
+        maximumFractionDigits: 4,
```

Comment:
This should probably go to the scientific advisory board, but I am strongly of the opinion that we display summary statistics to a number of **significant figures**.    Individual data points should be displayed an appropriate number of **decimal places** where appropriateness is related to the level of precision the datum can be measured.  Maybe we can come back to this when we have precision metadata from the back end.

As is:
![image](https://user-images.githubusercontent.com/308639/125334792-b2fa6380-e343-11eb-8fab-7e3878d8e592.png)

With maximumSignificantDigits: 4
![image](https://user-images.githubusercontent.com/308639/125334703-94946800-e343-11eb-9da5-79acc2a85d0d.png)

Reply:
I'm fine with whatever folks want. I was under the impression that the `precision` field will describe the number of decimals to show, not _actual_ precision. We should clarify.